src/sdk/pynni/nni/nas/pytorch/callbacks.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+
+_logger = logging.getLogger(__name__)
+
+
+class Callback:
+
+    def __init__(self):
+        self.model = None
+        self.mutator = None
+        self.trainer = None
+
+    def build(self, model, mutator, trainer):
+        self.model = model
+        self.mutator = mutator
+        self.trainer = trainer
+
+    def on_epoch_begin(self, epoch):
+        pass
+
+    def on_epoch_end(self, epoch):
+        pass
+
+    def on_batch_begin(self, epoch):
+        pass
+
+    def on_batch_end(self, epoch):
+        pass
+
+
+class LearningRateScheduler(Callback):
+    def __init__(self, scheduler, mode="epoch"):
+        super().__init__()
+        assert mode == "epoch"
+        self.scheduler = scheduler
+        self.mode = mode
```

Comment:
It looks wierd to pass one scheduler to another scheduler, can we rename the  LearningRateScheduler to include 'Callback' in its name?

Reply:
ok. changed.